src/openrct2/actions/RideEntranceExitPlaceAction.hpp

```diff
@@ -105,7 +105,7 @@ DEFINE_GAME_ACTION(RideEntranceExitPlaceAction, GAME_COMMAND_PLACE_RIDE_ENTRANCE
             return MakeResult(GA_ERROR::NOT_OWNED, errorTitle);
         }
 
-        int8_t clear_z = (z / 8) + (_isExit ? 5 : 7);
+        auto clear_z = (z / 8) + (_isExit ? 5 : 7);
```

Comment:
Make the same changes to L176 for this to actually work.

Reply:
It _does_ work, as I have tested it...